backend/init_app.py

```diff
@@ -29,6 +47,12 @@
 api.add_resource(Categories, '/api/category/<category_id>')
 api.add_resource(SubcategoryList, '/api/subcategory')
 api.add_resource(Subcategories, '/api/subcategory/<subcategory_id>')
+api.add_resource(UserRegistration, '/api/registration')
+api.add_resource(UserLogin, '/api/login')
+api.add_resource(UserLogoutAccess, '/api/logout/access')
```

Comment:
`/api/logout/access` -  what a strange URL :D What is it supposed to mean? If it's to logout a user, then POST on `/api/logout/` is enough

Reply:
I have to add refresh token and access token to backlist. But Refresh tokens cannot access an endpoint that is protected with jwt_required() and access tokens cannot access and endpoint that is protected with jwt_refresh_token_required() so I need 2 endpoints one for logout acccess token and one for logout refresh token.